md/bonita-layout.md

```diff
@@ -0,0 +1,9 @@
+# Bonita Layout
```

Comment:
I think this page needs a sentence explaining what is Bonita layout

Reply:
done